microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,575 @@
+---
+title: Get started with WebView2 in HoloLens 2 Unity apps
+description: Get started with WebView2 in HoloLens 2 Unity apps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/01/2023
+---
+# Get started with WebView2 in HoloLens 2 Unity apps (Preview)
+
+> [!NOTE]
+> WebView2 on HoloLens 2 and the WebView Unity plugin are both in Preview and are subject to change before general availability.
+
+In this tutorial, you:
+
+* Set up your development tools for creating HoloLens 2 Unity apps that use WebView2 to display web content.
+* Install the Mixed Reality Toolkit (MRTK) using the Mixed Reality Feature Tool.
```

Comment:
techreview: @michaelfarnsworth - in this .md file, Find "Mixed Reality Feature Tool", see if it should say "Mixed Reality Toolkit (MRTK)" instead.  

I globally changed from the hybrid combination:
MRTK Feature Tool
to:
Mixed Reality Feature Tool

I didn't change any to:
Mixed Reality Toolkit (MRTK)

Reply:
Looks good.

Reply:
you can close this thread